hypervisor/arch/x86/assign.c

```diff
@@ -69,7 +69,7 @@ _lookup_entry_by_id(uint32_t id)
 }
 
 static inline struct ptdev_remapping_info *
-lookp_entry_by_id(uint32_t id)
+lookup_entry_by_id(uint32_t id)
```

Comment:
Good Catch @zhijianli88 
But, why this is an orphan API ?  @gzhai7

Reply:
Sorry. It's a mistake when spliting the big patches. This orphan API will be removed by the 2nd patches.  I'll continue to split the 2nd into small ones to address this.

Leave it as is or delete it. I'll handle it in next small patches.

Reply:
In this case, @zhijianli88 please delete the API.